docs/spec/concepts.rst

```diff
@@ -3,6 +3,268 @@
+represented by ``tuple[int, Any]``.
+
+The gradual guarantee
+~~~~~~~~~~~~~~~~~~~~~
+
+:ref:`Any` allows gradually adding static types to a dynamically-typed program.
+For a fully dynamically-typed program, a static checker has the type :ref:`Any`
+for all expressions, and should emit no errors. Adding static type annotations
+to the program (making the program more statically typed) may result in static
+type errors, if the program is not correct or if the static type annotations
+aren't able to fully represent the runtime types. Removing static type
+annotations (making the program more dynamic) should not result in additional
+static type errors. This is often referred to as the **gradual guarantee**.
+
+In Python's type system, we don't take the gradual guarantee as a strict
+requirement, but it's a useful guideline.
+
+Subtype, supertype, and type equivalence
+----------------------------------------
+
+A fully static type ``B`` is a **subtype** of another fully static type ``A``
+if and only if the set of values represented by ``B`` is a subset of the set of
+values represented by ``A``. Because the subset relation on sets is transitive
+and reflexive, the subtype relation is also transitive (if ``C`` is a subtype
+of ``B`` and ``B`` is a subtype of ``A``, then ``C`` is a subtype of ``A``) and
+reflexive (``A`` is always a subtype of ``A``).
+
+The **supertype** relation is the inverse of subtype: ``A`` is a supertype of
+``B`` if and only if ``B`` is a subtype of ``A``; or equivalently, if and only
+if the set of values represented by ``A`` is a superset of the values
+represented by ``B``. The supertype relation is also transitive and reflexive.
+
+We also define an **equivalence** relation on fully static types: the types
+``A`` and ``B`` are equivalent (or "the same type") if and only if ``A`` is a
+subtype of ``B`` and ``B`` is a subtype of ``A``. This means that the set of
+values represented by ``A`` is both a superset and a subset of the values
+represented by ``B``, meaning ``A`` and ``B`` must represent the same set of
+values.
+
+We may describe a type ``B`` as "narrower" than a type ``A`` (or as a "proper
+subtype" of ``A``) if ``B`` is a subtype of ``A`` and ``B`` is not equivalent
+to ``A``. In the same scenario we can describe the type ``A`` as "wider" than
+``B``, or a "proper supertype" of ``B``.
+
+Materialization
+---------------
+
+Since :ref:`Any` represents an unknown static type, it does not represent any
+known single set of values (it represents an unknown set of values.) Thus it is
+not in the domain of the subtype, supertype, or equivalence relations on static
+types described above.
+
+To relate gradual types more generally, we define a **materialization**
+relation. The intuition for materialization is that it transforms a "more
+dynamic" type to a "more static" type. Given a gradual type ``A``, if we
+replace zero or more occurrences of ``Any`` in ``A`` with some gradual type
```

Comment:
Why is passive voice preferable to "we"?

Reply:
Who is "we" here? ;)

The general tone of this article is quite formal, and the general advice given by style guides is to avoid first-person pronouns in this kind of formal writing. To me this wouldn't be out of place in a tutorial, or some other kind of article that addresses the reader constantly. In a formal specification that aims to describe abstract concepts in an objective way, using "we" here feels out of place to me.

Reply:
There are many, many similar uses of "we" in this article ("We will refer...", "we can assign", "we formalize...", "we don't take" all occur within the first twenty lines, where I stopped my search.) I feel like this is quite common in presentations of systems similar to this one.

Reply:
Alright, fair enough :)